### ISSUE TITLE
[Bug][MC] Broken textures and crashing near IC FC Objects

### ISSUE THREAD

**Issue by cm8263** [inferno-collection/Fire-Alarm#3]
**Describe the bug**

Tree textures go pink (Dev Texture), and MP ped hair turns green.

**Steps to reproduce**
Steps to reproduce the behavior.

Unknown.

**Expected behavior**

No texture issues.

**Screenshots**

![image](https://user-images.githubusercontent.com/10535902/64866417-c02e6480-d67e-11e9-9210-0d0bc6396272.png)
![image](https://user-images.githubusercontent.com/10535902/64866427-c58baf00-d67e-11e9-87d3-2b4e7270f7da.png)

**Resource version**
What resource version are you running, and have you tried the latest version?

Public Beta Version 4.45

**Additional context**

Mission Critical.

**Comment by cm8263** [comment]
**Reproduction Attempt 1**

Server:
`localhost`

Resource Version:
`Public Beta Version 4.45 Beta`

Resources running:
```
start mapmanager
start chat
start spawnmanager
start sessionmanager
start fivem
start hardcap
start rconlog
start scoreboard
start playernames
start lux_vehcontrol
start inferno-loading
start inferno-cars
start inferno-fire-alarm
start carcommand
start vMenu
```

Results:

- No issues joining server
- No errors in client log
- No errors in server log
- Objects spawned with no issues
- Resource performed as expected

Conclusion:
Reproduction Attempt 1 unsuccessfull, unable to reproduce bug.

**Comment by cm8263** [comment]
**Reproduction Attempt 2**

Difference from Reproduction Attempt 1:
Started the `inferno-objects` resource as well as `inferno-fire-alarm` (not recommended, however, not believed to be fatal).

Server:
`localhost`

Cache cleared before server start:
Yes

Resource Version:
`Public Beta Version 4.45 Beta`

Resources running:
```
start mapmanager
start chat
start spawnmanager
start sessionmanager
start fivem
start hardcap
start rconlog
start scoreboard
start playernames
start lux_vehcontrol
start inferno-loading
start inferno-cars
start inferno-fire-alarm
start inferno-objects
start carcommand
start vMenu
```

Results:

- No issues joining server
- No errors in client log
- No errors in server log
- Objects spawned with no issues
- Resource performed as expected

Conclusion:
Reproduction Attempt 2 unsuccessful, unable to reproduce bug.

**Comment by cm8263** [comment]
Fixed in Public Beta Version 4.46 (https://github.com/inferno-collection/Fire-Alarm/commit/c06926f11ec9a7eeeaebfb95aa246e74ad9ef0f5).
